src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedIfDepthCheck.java

```diff
@@ -42,6 +42,22 @@
  * <pre>
  * &lt;module name=&quot;NestedIfDepth&quot;/&gt;
  * </pre>
+ * <p>Valid Code Example:</p>
+ * <pre>
+ * if (true) {
+ *     if (true) {} // OK
+ *     else {}
+ * }
+ * </pre>
+ * <p>Invalid Code Example:</p>
```

Comment:
`Invalid Code Example:` -> `Invalid code example:`
Same for Valid code example.

Reply:
done

Reply:
resolved